### PR TITLE
Okello sam21 patch 1

### PR DESCRIPTION
**Pull request by OkelloSam21**

### CODE REVIEW

**Comment by adityaraute** [comment]
Hi @OkelloSam21

Not sure what this PR is as no file contents are changed here. Please mention the issue number in the comment or title of the PR as well, when you commit the necessary changes.

Marking the PR as draft for now.

**Comment by adityaraute** [comment]
Hi @OkelloSam21 ,

Make sure that for the facebook icon you download the svg file which is in the same theme as the other icons.

File: index.html [hunk 1]
```diff
@@ -54,10 +54,12 @@ <h1 class = "title translate" aria-label="रणजीत बख्शी जन
 
 
                     <a href="#"><img src="./assets/images/instagram.svg" alt="instagram"></a>
+                    <a href="https://www.facebook.com/akil.bakhshi/"><img src="./assets/images/faccebook.svg" alt="facebook"></a>
```
Comment (adityaraute) [inline index.html:57]:
```suggestion
                    <a href="https://www.facebook.com/akil.bakhshi/"><img src="./assets/images/facebook-f.svg" alt="facebook"></a>
```

**Review by adityaraute — CHANGES_REQUESTED** [review]
Looks like you didn't test it

**Review by adityaraute — CHANGES_REQUESTED** [review]
image


This is still not working. Please make the necessary change as I had suggested in my previous review.

**Comment by adityaraute** [comment]
If you're still not sure what change to make, please ask.

**Comment by OkelloSam21** [comment]
> If you're still not sure what change to make, please ask.
Where should I change?

**Comment by adityaraute** [comment]
> > If you're still not sure what change to make, please ask.
> > Where should I change?

Your svg filename is facebook-f.svg
HTML href name is faccebook.svg

That's causing the issue? make them both the same.

**Comment by OkelloSam21** [comment]
> > > If you're still not sure what change to make, please ask.
> > > Where should I change?
> 
> Your svg filename is facebook-f.svg HTML href name is faccebook.svg
> 
> That's causing the issue. Make them both the same.
here is my output...
RBJKS__Home - Google Chrome 3_7_2023 10_52_38 PM (2)

**Comment by OkelloSam21** [comment]
I think am done

**Review by OkelloSam21 — COMMENTED** [review]
Completed

**Comment by adityaraute** [comment]
image


What do you mean your done? The href and the filename still do not match. Your svg file is named facebook-f.svg. Change it to facebook.svg and then it will work.

image

I'm still seeing the error, are you not testing the changes on your end?

**Review by OkelloSam21 — COMMENTED** [review]
renamed Facebook-f.svg to Facebook.svg

File: index.html (hunk 1 above)
Comment (adityaraute) [inline index.html:57]:
```suggestion
                    <a href="https://www.facebook.com/akil.bakhshi/"><img src="./assets/images/facebook.svg" alt="facebook"></a>
```

**Review by adityaraute — CHANGES_REQUESTED** [review]
It still doesn't work my friend
You spelt facebook wrong
image

image


I've made a suggestion, you can commit that.

**Comment by OkelloSam21** [comment]
> image
> 
> What do you mean your done? The href and the filename still do not match. Your svg file is named facebook-f.svg. Change it to facebook.svg and then it will work.
> 
> image
> 
> I'm still seeing the error, are you not testing the changes on your end?

I've committed your suggestion

**Comment by adityaraute** [comment]
Great!

Are you new to open Source?

**Comment by OkelloSam21** [comment]
> Great!
> 
> Are you new to Open Source?

Yeah...In fact, this was my first opensource contribution

**Comment by adityaraute** [comment]
Understandable.

**Pull request merged**